src/components/Navigation/Navigation.scss

```diff
@@ -257,23 +285,39 @@ $secondary-item-font-size: rem(15px);
   margin-bottom: spacing(tight);
   margin-left: nav(icon-size) + spacing(loose);
   overflow-x: var(--p-override-visible, hidden);
+
+  .Navigation-newDesignLanguage & {
+    margin-left: 0;
```

Comment:
We sub nav items to be full width now

![image](https://screenshot.click/DLPolaris_Web_Components__Figma_2020-02-28_10-42-17.png)